src/app/pages/class-details/class-details.view/__snapshots__/class-details.view.component.spec.ts.snap

```diff
@@ -40,6 +40,7 @@ exports[`ClassDetailsComponent should match current snapshot 1`] = `
             >
               <mat-form-field>
                 <input
+                  id="class-name"
```

Comment:
is this id is for testing purposes? if so can you please use `data-test-id="details-view-class-name-input"` instead